roles/georchestra/templates/datafeeder/datafeeder.service.j2

```diff
@@ -1,10 +1,10 @@
 [Unit]
-Description=datafeeder
+Description=geOrchestra datafeeder backend
 After=syslog.target
 
 [Service]
 User=www-data
-ExecStart=/usr/bin/java -Dserver.port={{ datafeeder.port }} -jar /srv/webapps/datafeeder.jar
+ExecStart=/usr/bin/java -Dserver.port={{ datafeeder.port }} -jar /usr/share/lib/georchestra-datafeeder/datafeeder-bin.jar
```

Comment:
I find the jar renaming quite curious, did I miss something in the geor/geor changes about it ?

Reply:
the build produces a small `datafeeder.jar` and a larger `datafeeder-bin.jar` which iirc is standalone.
```
-rw-r--r-- 1 landry users 91860381 Aug 24 12:16 datafeeder-bin.jar
-rw-r--r-- 1 landry users   890368 Aug 24 12:16 datafeeder.jar
```

from testing (cf georchestra/georchestra@b88c429) i had to use the large one.